### PR TITLE
186 remove contract upload

### LINKED ISSUE #186: Remove contract upload process (CONTRACT_UPLOADED and COMPLETED status)
Right now, in Credere, for FIs, we only have one "approval" step and then only the option to put the amount granted. But now, we found out that they are not allowed to approve or even say they are pre-approving a credit through Credere.  We also learned that after reviewing the documents and application sent through Credere, the FI still need to ask for more documents and internal process before approving or denying the credit. 
For the pilot, our workaround is to: 
- Edit the "approval" email text (https://github.com/open-contracting/credere-backend/commit/17a68405d27d9a1c78ba1f41faebd8b00ed12551) to indicate that only the documents were reviewed but there is still an onboarding to do with the FI directly. 
- After the application has been "approved" via Credere, but then the internal processes and reviews fail and they have to reject the application, ask FIs to put "0" as the amount granted and not send any more emails to the MSMEs, as the FI is communicating with the MSME directly anyway. (to be implemented)

In the future, we should implement a pre-approval step fully in Credere and remove this workaround.

### PR DESCRIPTION
**Pull request by yolile**
closes #186

### CODE REVIEW

File: docs/_static/routes.csv [hunk 1]
```diff
@@ -1,56 +1,55 @@
-Methods,Path,Backend parsers,Backend serializers,Frontend request,Frontend response
```
Comment (jpmckinney) [inline docs/_static/routes.csv:1]: Why did you delete some of the columns? Also `upload-contract` is still in the table.

File: migrations/versions/59cf8dc3cfde_.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+"""
```
Comment (jpmckinney) [inline migrations/versions/59cf8dc3cfde_.py:1]: This migration does nothing. Delete it.

**Review by jpmckinney — COMMENTED** [review]
Sweet. Nice PR to review before the weekend :) Just a couple comments.

File: migrations/versions/59cf8dc3cfde_.py (hunk 2 above)
Comment (yolile) [reply migrations/versions/59cf8dc3cfde_.py:1]: It was needed after the merge with your migration file from #429, because it had two HEADS after it, so I had to run `alembic merge heads` https://alembic.sqlalchemy.org/en/latest/branches.html

File: docs/_static/routes.csv (hunk 1 above)
Comment (yolile) [reply docs/_static/routes.csv:1]: Hmm I ran `python -m app dev routes --csv-format` to generate it as per documented, maybe I missed something?

Comment (jpmckinney) [reply docs/_static/routes.csv:1]: Ah, those docs and/or the command needs to be updated to preserve the content. I suggest copying the file from `main` and just deleting the rows that are out-of-date.

Comment (jpmckinney) [reply docs/_static/routes.csv:1]: I fixed the command on main. You can merge main, copy over routes.csv, and run:

```
python -m app dev routes --csv-format --file docs/_static/routes.csv
```

Comment (yolile) [reply docs/_static/routes.csv:1]: > and just deleting the rows that are out-of-date.

And then this?

Comment (jpmckinney) [reply docs/_static/routes.csv:1]: No longer needed, I don't think.

**Review by jpmckinney — APPROVED** [review]
